From db3546f6234506d51caf91e3763bfdf000828156 Mon Sep 17 00:00:00 2001
From: Henrik Riomar <henrik.riomar@gmail.com>
Date: Fri, 23 Nov 2018 07:40:22 +0100
Subject: [PATCH] systemLinux.c: fixup for Alpine

/sbin/shutdown -r now -> /sbin/reboot
/sbin/shutdown -h now -> /sbin/halt
---
 open-vm-tools/lib/system/systemLinux.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/open-vm-tools/lib/system/systemLinux.c b/open-vm-tools/lib/system/systemLinux.c
index c74d4890..c98ce962 100644
--- a/open-vm-tools/lib/system/systemLinux.c
+++ b/open-vm-tools/lib/system/systemLinux.c
@@ -312,7 +312,7 @@ System_Shutdown(Bool reboot)  // IN: "reboot or shutdown" flag
 #elif defined(USERWORLD)
       cmd = "/bin/reboot";
 #else
-      cmd = "/sbin/shutdown -r now";
+      cmd = "/sbin/reboot";
 #endif
    } else {
 #if __FreeBSD__
@@ -322,7 +322,7 @@ System_Shutdown(Bool reboot)  // IN: "reboot or shutdown" flag
 #elif defined(USERWORLD)
       cmd = "/bin/halt";
 #else
-      cmd = "/sbin/shutdown -h now";
+      cmd = "/sbin/halt";
 #endif
    }
    if (system(cmd) == -1) {
-- 
2.19.1

